examples/server-history.py

```diff
@@ -34,12 +42,17 @@
 
     # enable history for this particular node, must be called after start since it uses subscription
     server.iserver.enable_history(myvar, period=None, count=100)
+
+    # enable history for myobj events generated by myevgen
+    server.iserver.enable_event_history(myobj, myevgen.event, period=None)
```

Comment:
I just checked the spec. user can specify what eventtype they want to read from. so the argument should be a list of event types, not a list of properties. Then internally we get all properties for every event type